scripts/jobrunner.sh

```diff
@@ -9,6 +9,7 @@ umask 027
 
 # ensure shared user is set up properly
 id -u jobrunner >/dev/null 2>&1 || useradd jobrunner --create-home --shell /bin/bash -G docker
+usermod -a -G docker jobrunner
```

Comment:
not sure why we need this, `useradd` already has `-G docker` that should do the same thing?

Reply:
so it does!

Reply:
ah I remember why I did this - some existing backends may have had their `jobrunner` user created without the `docker` group being added, so this fixes them

Reply:
ah you're right! if the `jobrunner` user already exists.